src/main/java/plugily/projects/murdermystery/arena/ArenaManager.java

```diff
@@ -321,6 +322,8 @@ public static void leaveAttempt(Player player, Arena arena) {
     }
     player.setGlowing(false);
     user.setSpectator(false);
+  	if(arena.isDeathPlayer(player)) arena.removeDeathPlayer(player);
+    if(arena.isSpectatorPlayer(player)) arena.removeSpectatorPlayer(player);
```

Comment:
```suggestion
if (arena.isDeathPlayer(player)) {
  arena.removeDeathPlayer(player);
}

if (arena.isSpectatorPlayer(player)) {
  arena.removeSpectatorPlayer(player);
}

```
Keep the code readable